apps/api/src/server/utils/mapping/map-document-details.js

```diff
@@ -1,36 +1,104 @@
 import { mapDateStringToUnixTimestamp } from './map-date-string-to-unix-timestamp.js';
 
-/** @typedef {import('@pins/api').Schema.Document} Document */
+/** @typedef {import('apps/api/prisma/schema.js').DocumentFullMetadata} DocumentFullMetadata */
+/** @typedef {import('apps/api/prisma/schema.js').IDocument} Document */
 /** @typedef {import('apps/api/prisma/schema.js').DocumentDetails} DocumentDetails */
 
 /**
  *
- * @param { Document } documentDetails
+ * @param { DocumentFullMetadata } documentMetadata
  * @returns { DocumentDetails }
  */
-export const mapSingleDocumentDetails = (documentDetails) => {
-	// TODO: currently some fields not in DB, hence null / blank values returned
+export const mapSingleDocumentDetailsFromMetadata = ({ Document, ...documentMetadata }) => {
+	// Published date
 	return {
-		guid: documentDetails.guid,
-		documentName: documentDetails.name,
-		blobStorageContainer: documentDetails.blobStorageContainer,
-		blobStoragePath: documentDetails.blobStoragePath,
-		from: '',
-		receivedDate: mapDateStringToUnixTimestamp(documentDetails.createdAt.toString()),
-		size: documentDetails.fileSize,
-		type: documentDetails.fileType ?? '',
-		redacted: documentDetails.redacted,
-		status: documentDetails.status,
-		description: '',
-		documentReferenceNumber: '',
-		version: 1,
-		agent: '',
-		caseStage: '',
-		webFilter: '',
-		documentType: ''
+		documentGuid: documentMetadata.documentGuid,
```

Comment:
hey, I tried and seems like this line creates an error.
Spefically the API _/applications/{caseId}/folders/{folderId}/documents_ returns this error:

```
{
  "errors": "Cannot read properties of null (reading 'documentGuid')"
}
```

do you know what could cause it?